### PR TITLE
issue 424 : textual output small fix

### LINKED ISSUE #424: textual output small fix
`list` output does not end with `\n` ... please check additional commands output, and add newline for all textual output.
example:
```
<prompt>$ ./bin/k8snetpolicy list --dirpath tests/acs-security-demos
backend/checkout[Deployment] => backend/notification[Deployment] : TCP 8080
backend/checkout[Deployment] => backend/recommendation[Deployment] : TCP 8080
backend/checkout[Deployment] => payments/gateway[Deployment] : TCP 8080
backend/recommendation[Deployment] => backend/catalog[Deployment] : TCP 8080
backend/reports[Deployment] => backend/catalog[Deployment] : TCP 8080
backend/reports[Deployment] => backend/recommendation[Deployment] : TCP 8080
frontend/webapp[Deployment] => backend/checkout[Deployment] : TCP 8080
frontend/webapp[Deployment] => backend/recommendation[Deployment] : TCP 8080
frontend/webapp[Deployment] => backend/reports[Deployment] : TCP 8080
frontend/webapp[Deployment] => backend/shipping[Deployment] : TCP 8080
payments/gateway[Deployment] => payments/mastercard-processor[Deployment] : TCP 8080
payments/gateway[Deployment] => payments/visa-processor[Deployment] : TCP 8080
{ingress-controller} => frontend/asset-cache[Deployment] : TCP 8080
{ingress-controller} => frontend/webapp[Deployment] : TCP 8080<prompt>$
```

last lines should be  instead:
```
{ingress-controller} => frontend/webapp[Deployment] : TCP 8080
<prompt>$
```

### PR DESCRIPTION
**Pull request by shireenf-ibm**
#424
- adding new line to textual outputs that did not have (connlist + diff);
- tests of list with exposure-analysis (with non empty result) had new line at the end
- added new line to case of empty exposure analysis section (test run with `--exposure` but there is no exposure)

### CODE REVIEW

File: pkg/netpol/connlist/conns_formatter_txt.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ type formatText struct {
 func (t *formatText) writeOutput(conns []Peer2PeerConnection, exposureConns []ExposedPeer, exposureFlag bool) (string, error) {
 	res := t.writeConnlistOutput(conns, exposureFlag)
 	if !exposureFlag {
-		return res, nil
+		return res + newLineChar, nil
```
Comment (adisos) [inline pkg/netpol/connlist/conns_formatter_txt.go:25]: better place the addition of the `newLineChar`  where `Join` is called, within `writeConnlistOutput`, as this is directly related to the result from `join`, where it does not add newline for that result.

File: pkg/netpol/connlist/conns_formatter_txt.go [hunk 2]
```diff
@@ -60,6 +60,9 @@ func (t *formatText) writeExposureOutput(exposureResults []ExposedPeer) string {
 	sort.Strings(unprotectedLines)
 	// writing results of exposure for all peers
 	res := exposureAnalysisHeader
+	if len(egressExpLines) == 0 && len(ingressExpLines) == 0 {
+		return res + newLineChar
```
Comment (adisos) [inline pkg/netpol/connlist/conns_formatter_txt.go:64]: shouldn't `exposureAnalysisHeader` be always with newline already??

**Review by adisos — COMMENTED** [review]
added few comments.
please also check `md,csv` output

File: pkg/netpol/connlist/conns_formatter_txt.go (hunk 2 above)
Comment (shireenf-ibm) [reply pkg/netpol/connlist/conns_formatter_txt.go]: true, had the new line char at the beginning of the new sub-section; fixed

File: pkg/netpol/connlist/conns_formatter_txt.go (hunk 1 above)
Comment (shireenf-ibm) [reply pkg/netpol/connlist/conns_formatter_txt.go]: done

**Review by adisos — APPROVED** [review]

**Pull request merged**